SQLAssignment.sql.sql

```diff
@@ -0,0 +1,1043 @@
++------+----------+---------+--------+------+
+| 2008 | Cisneros | SanJose |    300 | 1007 |
+| 2003 | Liu      | SanJose |    200 | 1002 |
++------+----------+---------+--------+------+
+10) Match salespeople to customers according to what city they live in.
+mysql> SELECT s.SNAME,c.CNAME,s.CITY FROM Customers c,Salespeople s WHERE s.CITY = c.CITY;
++--------+----------+---------+
+| SNAME  | CNAME    | CITY    |
++--------+----------+---------+
+| Fran   | Hoffman  | London  |
+| Motika | Hoffman  | London  |
+| Peel   | Hoffman  | London  |
+| Serres | Liu      | SanJose |
+| Fran   | Clemens  | London  |
+| Motika | Clemens  | London  |
+| Peel   | Clemens  | London  |
+| Serres | Cisneros | SanJose |
++--------+----------+---------+
+11) Find all the customers in SanJose who have a rating above 200.
+
+mysql> SELECT CNUM,CNAME,CITY,RATING,SNUM FROM Customers WHERE CITY='SanJose'AND RATING>=200;
++------+----------+---------+--------+------+
+| CNUM | CNAME    | CITY    | RATING | SNUM |
++------+----------+---------+--------+------+
+| 2003 | Liu      | SanJose |    200 | 1002 |
+| 2008 | Cisneros | SanJose |    300 | 1007 |
++------+----------+---------+--------+------+
+12) List the names and commissions of all salespeople in London.
+mysql> SELECT DISTINCT SNUM,CITY,COMM FROM Salespeople WHERE CITY ='London';
++------+--------+------+
+| SNUM | CITY   | COMM |
++------+--------+------+
+| 1001 | London |   12 |
+| 1004 | London |   11 |
+| 1008 | London |   25 |
++------+--------+------+
+13) List all the orders of Salesperson Motika from the orders table.
+mysql> SELECT o.ONUM,o.AMT,o.ODATE,o.CNUM,s.SNUM,s.SNAME FROM Orders o,Salespeople s WHERE s.SNAME = 'Motika'AND s.SNUM = o.SNUM;
++------+---------+------------+------+------+--------+
+| ONUM | AMT     | ODATE      | CNUM | SNUM | SNAME  |
++------+---------+------------+------+------+--------+
+| 3002 | 1900.10 | 1990-10-03 | 2007 | 1004 | Motika |
++------+---------+------------+------+------+--------+
+
+14) Find all customers who booked orders on October 3.
+
+
+
+mysql>  SELECT c.CNUM,c.CNAME,c.CITY,c.RATING,c.SNUM,o.ODATE FROM Customers c,Orders o WHERE o.ODATE='1990/10/03' AND c.CNUM = O.CNUM;
++------+----------+---------+--------+------+------------+
+| CNUM | CNAME    | CITY    | RATING | SNUM | ODATE      |
++------+----------+---------+--------+------+------------+
+| 2001 | Hoffman  | London  |    100 | 1001 | 1990-10-03 |
+| 2003 | Liu      | SanJose |    200 | 1002 | 1990-10-03 |
+| 2008 | Cisneros | SanJose |    300 | 1007 | 1990-10-03 |
+| 2008 | Cisneros | SanJose |    300 | 1007 | 1990-10-03 |
+| 2007 | Pereira  | Rome    |    100 | 1004 | 1990-10-03 |
++------+----------+---------+--------+------+------------+
+
+
+15) Give the sums of the amounts from the Orders table, grouped by date, eliminating all those dates where the SUM was not at least 2000 above
+the maximum Amount.
+mysql> SELECT ODATE,SUM(AMT) AS Total_AMT FROM Orders GROUP BY ODATE;
```

Comment:
This is an incomplete query. Please add conditional check for more than 2000 sum.